phoenix-pherf/src/main/java/org/apache/phoenix/pherf/Pherf.java

```diff
@@ -0,0 +1,204 @@
+
+    public Pherf(String[] args) throws Exception {
+        CommandLineParser parser = new PosixParser();
+        CommandLine command = null;
+        HelpFormatter hf = new HelpFormatter();
+
+        try {
+            command = parser.parse(options, args);
+        } catch (ParseException e) {
+            hf.printHelp("Pherf", options);
+            System.exit(1);
+        }
+
+        properties = getProperties();
+        dropPherfTablesRegEx = command.getOptionValue("drop", null);
+        monitor = command.hasOption("m");
+        String monitorFrequency = (command.hasOption("m") && command.hasOption("monitorFrequency"))
+                ? command.getOptionValue("monitorFrequency")
+                : properties.getProperty("pherf.default.monitorFrequency");
+        properties.setProperty("pherf.default.monitorFrequency", monitorFrequency);
+
+        logger.debug("Using Monitor: " + monitor);
+        logger.debug("Monitor Frequency Ms:" + monitorFrequency);
+        loadData = command.hasOption("l");
+        executeQuerySets = command.hasOption("q");
+//        release = command.hasOption("r") ? command.getOptionValue("r") : null;
+        zookeeper = command.getOptionValue("z", "localhost");
+        queryHint = command.getOptionValue("hint", null);
+        exportCSV = command.hasOption("export");
+        diff = command.hasOption("diff");
+        listFiles = command.hasOption("listFiles");
+        scenarioFile = command.hasOption("scenarioFile") ? command.getOptionValue("scenarioFile") : null;
+        schemaFile = command.hasOption("schemaFile") ? command.getOptionValue("schemaFile") : null;
+        rowCountOverride = Integer.parseInt(command.getOptionValue("rowCountOverride", "0"));
+        String writerThreadPoolSize = command.getOptionValue("writerThreadSize",
+                properties.getProperty("pherf.default.dataloader.threadpool"));
+        properties.setProperty("pherf. default.dataloader.threadpool", writerThreadPoolSize);
+
+
+        if ((command.hasOption("h") || (args == null || args.length == 0))
+                && !command.hasOption("listFiles")) {
+            hf.printHelp("Pherf", options);
+            System.exit(1);
+        }
+        PhoenixUtil.setZookeeper(zookeeper);
+        PhoenixUtil.setRowCountOverride(rowCountOverride);
+        PhoenixUtil.writeSfdcClientProperty();
+    }
+
+    public static void main(String[] args) {
+        try {
+            new Pherf(args).run();
+        } catch (Exception e) {
+            e.printStackTrace();
+            System.exit(1);
+        }
+    }
+
+    public void run() throws Exception {
+        WorkloadExecutor workloadExec = null;
+        try {
+            if (listFiles) {
+                ResourceList list = new ResourceList(PherfConstants.RESOURCE_DATAMODEL);
+                Collection<Path> schemaFiles = list.getResourceList(PherfConstants.SCHEMA_ROOT_PATTERN + ".sql");
+                System.out.println("Schema Files:");
```

Comment:
Actually this is on purpose. We want to print to console the files that are available for use. It's part of the listFiles command.